docs/data/tree-view/datasets/employees.ts

```diff
@@ -0,0 +1,40 @@
+import { TreeViewBaseItem } from '@mui/x-tree-view/models';
+
+export const EMPLOYEES_DATASET: TreeViewBaseItem[] = [
```

Comment:
I recreated the exact same dataset as the one I did for the Tree Data feature of the grid :+1: